src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -240,14 +241,25 @@ private String fieldTypeCardinalityComment(Field field) {
   }
 
   /**
+   * The type of parameter.
+   */
+  private enum ParamType {
+    OPTIONAL,
+    REQUIRED,
+    ENCAPSULATED_REQUIRED
+  }
+
+  /**
    * Returns a JSDoc comment string for the field as a parameter to a function.
    */
-  private String fieldParamComment(Field field, String paramComment, boolean isOptional) {
+  private String fieldParamComment(Field field, String paramComment, ParamType paramType) {
     String commentType = fieldTypeCardinalityComment(field);
-    String fieldName = wrapIfKeywordOrBuiltIn(lowerUnderscoreToLowerCamel(field.getSimpleName()));
-    if (isOptional) {
+    String fieldName = lowerUnderscoreToLowerCamel(field.getSimpleName());
+    if (paramType == ParamType.OPTIONAL) {
```

Comment:
Oh, this is not my intention here -- I think all of the parameters are under `request.`. `options` will contain call options and that's it -- this is more usable.